hieradata/node/comcam-archiver.cp.lsst.org.yaml

```diff
@@ -1,105 +1,96 @@
 ---
-profile::core::nm_dispatch::interfaces:
-  em1:
-    - "/sbin/ethtool --set-ring ${DEV} rx 4096 tx 4096"
-    - "tc filter del dev ${DEV} chain 0 || true"
-    - "tc qdisc del dev ${DEV} root || true"
-    - "tc qdisc add dev ${DEV} root handle 1: htb default 1"
-    - "tc class add dev ${DEV} parent 1: classid 1:1 htb rate 10gbit burst 40mbit"
-    - "tc class add dev ${DEV} parent 1:1 classid 1:10 htb rate 2.5gbit burst 40mbit"
-    - "tc filter add dev ${DEV} parent 1: protocol ip matchall flowid 1:1"
-    - "tc filter add dev ${DEV} parent 1:1 protocol ip prio 1 u32 match ip dst 141.142.0.0/16 flowid 1:10"
-network::interfaces_hash:
-  em1:  # fqdn
-    bootproto: "dhcp"
-    defroute: "yes"
-    onboot: "yes"
-    type: "Ethernet"
-  em2:  # dds
-    bootproto: "none"
-    defroute: "no"
-    ipaddress: "139.229.166.1"
-    netmask: "255.255.255.0"
-    nozeroconf: "yes"
-    onboot: "yes"
-    type: "Ethernet"
-  em3:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  em4:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  p2p1:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  p2p2:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
+nm::connections:
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "b28d29b1-83cc-4df2-9c65-22cd9d97d0af"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        method: "auto"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  eno2:  #dds interface
```

Comment:
OSPL/DDS no longer runs on the -archiver hosts. I think we should disable the dds interface and free any route53 records.